lib/tasks/fetch_missing_metadata.rake

```diff
@@ -0,0 +1,41 @@
+require "gds_api/rummager"
+require "gds_api/publishing_api_v2"
+require "missing_metadata_fetcher"
+
+PAGE_SIZE = 1000
+MAX_PAGES = 52
+
+desc "Fetch missing document metadata from the publishing api"
+task :populate_metadata do
+  rummager = GdsApi::Rummager.new(Plek.new.find("rummager"))
+  publishing_api = GdsApi::PublishingApiV2.new(Plek.new.find("publishing-api"))
+  fetcher = MissingMetadataFetcher.new(publishing_api)
+
+  0.upto MAX_PAGES do |page|
+    response = rummager.search({
+      "filter_content_store_document_type" => "_MISSING",
+      "count" => PAGE_SIZE,
+      "start" => page * PAGE_SIZE,
```

Comment:
This is slightly dangerous as you are requesting page two after updated records on page 1. potentially it would be better to get all the pages first then process or process pages in reverse order.